README.md

```diff
@@ -48,22 +48,49 @@ We recommend downloading the latest application binary from [our Releases page](
 
 ## Running a development build
 
-First, [install Rust via `rustup`](https://www.rust-lang.org/tools/install). This project uses a lot of Rust compiled to [WASM](https://webassembly.org/) within it. We always use the latest stable version of Rust, so you may need to run `rustup update stable`. Then, run:
+Install a node version manager such as [fnm](https://github.com/Schniz/fnm?tab=readme-ov-#installation).
 
+On Windows, it's also recommended to upgrade your PowerShell version, we're using 7.
+
+Then in the repo run the following to install and use the node version specified in `.nvmrc`. You might need to specify your processor architecture with `--arch arm64` or `--arch x64` if it's not autodetected. 
+```
+fnm install --corepack-enabled
+fnm use
+```
+
+Install the NPM dependencies with:
 ```
 yarn install
 ```
 
-followed by:
+This project uses a lot of Rust compiled to [WASM](https://webassembly.org/) within it. We always use the latest stable version of Rust, so you may need to run `rustup update stable`. We have package scripts to run rustup, see `package.json` for reference:
```

Comment:
This isn't technically true. Although we stay up to date with the latest stable, the exact version is specified in `src/wasm-lib/rust-toolchain.toml`. Whether you run `rustup update stable` has no effect. Even when I've run it, cargo still seems to re-download for me. It hasn't been smart enough to see that they resolve to the same thing.

Reply:
But maybe it will affect cargo commands like `cargo install` when run from the root directory, not `src/wasm-lib`.

Reply:
Well it's probably time to remove it then, that was from main I believe.

Reply:
Updated